server/circleci.go

```diff
@@ -49,11 +49,17 @@ func (s *Server) triggerCircleCiIfNeeded(ctx context.Context, pr *model.PullRequ
 		return
 	}
 
+	blockList := s.Config.BlacklistPaths
+	repoBlockList, ok := s.Config.RepoBlockListPaths[pr.RepoName]
+	if ok {
+		blockList = append(blockList, repoBlockList...)
```

Comment:
:thinking: Can I also get privately messaged the list of files please?

Reply:
You mean the new list?  See https://github.com/mattermost/platform-private/pull/265 for that.